packages/pages/src/vite-plugin/build/buildStart/rendering/wrapper.ts

```diff
@@ -1,26 +1,29 @@
 import { TemplateModuleInternal } from "../../../../common/src/template/internal/types.js";
-import {
-  renderHeadConfigToString,
-  getLang,
-} from "../../../../common/src/template/head.js";
+import { getLang } from "../../../../common/src/template/head.js";
 import {
   TemplateRenderProps,
-  GetHeadConfig,
+  RenderTemplate,
 } from "../../../../common/src/template/types.js";
+import {
+  getHydrationTemplate,
+  getServerTemplatePlugin,
+} from "../../../../common/src/template/hydration.js";
+import path from "path";
 
-export const reactWrapper = <T extends TemplateRenderProps>(
+export const reactWrapper = async <T extends TemplateRenderProps>(
   props: T,
   templateModuleInternal: TemplateModuleInternal<any, any>,
-  template: string,
   hydrate: boolean,
```

Comment:
unused

Reply:
Good call. I left it there for when it's configurable. I've updated to actually use the boolean now, but it's still hardcoded to true further up. Once we make it configurable though it'll just work.